azure-pipelines.yml

```diff
@@ -362,20 +368,48 @@ stages:
           steps:
           - checkout: self
             clean: true
-          - script: ./fcs/build.sh
-            displayName: Build
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core sdk'
+            inputs:
+              useGlobalJson: true
+              workingDirectory: fcs
+          - script: ./fcs/build.sh Test
+            displayName: Build / Test
+          - task: PublishTestResults@2
+            displayName: Publish Test Results
+            inputs:
+              testResultsFormat: 'NUnit'
+              testResultsFiles: '*.xml'
+              searchFolder: '$(Build.SourcesDirectory)/artifacts/TestResults/Release'
+              publishRunAttachments: true
+            continueOnError: true
+            condition: always()
 
-        # - job: MacOS_FCS
-        #   pool:
-        #     vmImage: macOS-latest
-        #   variables:
-        #   - name: _SignType
-        #     value: Test
-        #   steps:
-        #   - checkout: self
-        #     clean: true
-        #   - script: ./fcs/build.sh
-        #     displayName: Build
+        - job: MacOS_FCS
```

Comment:
@brettfo  do we want to enable this?

Reply:
I enabled them to prove that the hard-crashes are gone. We can re-comment them knowing that net4x FCS is horribly broken, and then I can re-renable them in the PR dedicated to greening them up.

Reply:
If I recall, the mac FCS build was turned off due to a mono bug, but I have no idea if it was the bug described earlier or not.

If this or any PR can show that the mac FCS leg will pass, I'm fine to re-enable it.

Reply:
The problem right now is that the FCS legs _will not pass_ the testing stage, because the reference assembly resolution has drifted on those platforms.  I can change the step to just do the build for this PR, which should work, but doesn't actually tell us much of anything, because the failures happened during test runtime.